CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - PR #538 Upgrade CUB and Thrust to the latest commits
 - PR #542 Pin conda spdlog versions to 1.7.0
 - PR #550 Remove CXX11 ABI handling from CMake
+- PR #578 Switch thrust to use the NVIDIA/thrust repo
```

Comment:
Why make this so out of order?

Reply:
I just inserted the line randomly to avoid merge conflict. :) Should they be sorted?

Reply:
Seems nicer sorted, but don't worry about it.